dhall/src/Dhall/Main.hs

```diff
@@ -394,6 +407,12 @@ command (Options {..}) = do
             putStrLn dhallVersionString
 
         Default {..} -> do
+            if version
+                then do
+                    putStrLn dhallVersionString
+                    Exit.exitSuccess
```

Comment:
I believe you don't want to `exitSuccess` here because `handle` will catch and display the `ExitSuccess` error

Reply:
`handle` delegates to `handleExitCode`:

```
            handleExitCode e = do
                Control.Exception.throwIO (e :: ExitCode)
```

which is just the same as `System.Exit.exitWith`.

Reply:
Oh, you're right!  Never mind, then